plugins/mcreator-localization/help/zh_cn/biome/generate_mineshafts.md

```diff
@@ -0,0 +1 @@
+Select this setting to have mineshafts in your biome.
```

Comment:
Do not include tool tips that are not translated. CMreator will automatically use the default (English) ones.

Reply:
i know, this just i put the files in the folder